frontend/src/Commons/Forms/InputModules/Summary/Summary.tsx

```diff
@@ -12,43 +12,70 @@ import InputModuleProps from 'src/Interface/InputModuleProps';
 import InputModuleInterface from 'src/Interface/InputModuleinterface';
 import { faList } from '@fortawesome/free-solid-svg-icons';
 import CustomLink from '@Commons/CustomLink/CustomLink';
-import { StyledFarmInfo } from './Summary.style';
+import { StyledFarmInfo, StyledContainer } from './Summary.style';
 
-const SummaryComponent: React.FC<InputModuleProps> = ({ farmDetails }) => (
-  <>
-    <StyledFarmInfo>
-      <p>
-        Farm Name:&nbsp;
-        {farmDetails.FarmName}
-      </p>
-      <p>
-        Year:&nbsp;
-        {farmDetails.Year}
-      </p>
-      <p>
-        Region:&nbsp;
-        {farmDetails.FarmRegion}
-      </p>
-      <p>
-        Berries:&nbsp;
-        {farmDetails.HasBerries ? 'Yes' : 'No'}
-      </p>
-      <p>Fields:&nbsp;</p>
-      {farmDetails.Fields.map((field) => (
-        <div key={field.FieldName}>
-          <p>{field.FieldName}</p>
-          <p>{field.Area}</p>
-          <p>{field.Comments}</p>
+const SummaryComponent: React.FC<InputModuleProps> = ({ farmDetails }) => {
+  function isCommentEmpty(comment: string | null | undefined): boolean {
+    return comment === '' || comment === null;
```

Comment:
In Javascript you can use strings as falsey and truthy, eliminating the need for this function
https://pinta.land/posts/js-truthy-falsy/#truthy-values